docs/astro/sql/operators/dataframe.rst

```diff
@@ -0,0 +1,20 @@
+
+.. dataframe_operator:
+
+==================
+dataframe operator
+==================
+
+.. _dataframe_operator:
```

Comment:
Do you intend to have this same line on L2?

Reply:
If so, we should remove this.

```suggestion
```

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/631/commits/46ed25c9f06bf5a843281352718547b3dc5f47f2)